assets/findify-autocomplete.js

```diff
@@ -110,41 +110,51 @@ const initAutocomplete = () => {
           document.querySelector('.findify-autocomplete').className += ' hidden';
         }
     };
-
+  
     const openAutocomplete = () => {
         const autocompleteClassName = document.querySelector('.findify-autocomplete').className;
         document.querySelector('.findify-autocomplete').className = autocompleteClassName.replace(' hidden', '');
         document.querySelector('.findify-close-autocomplete')?.addEventListener("click", (e) => closeAutocomplete(e));
-        setViewAll();
+        setTitle();
```

Comment:
Same as comment below.
We have all data needed to do conditions directly on .liquid side